run_benchmark_multi.sh

```diff
@@ -16,15 +16,14 @@ hash=$(git rev-parse HEAD)
 make build-headless
 
 rm -rf "$work_dir/temp/"; mkdir "$work_dir/temp/"
-docker container stop garage_benchmark
-docker container rm garage_benchmark
+docker container stop garage_benchmark || true && docker rm garage_benchmark || true
 for FILE in "$work_dir/garage/tests/benchmarks/"*
     do
             testname=`basename $FILE .py`
             if [[ $testname == "test"* ]]
             then
                 echo "running test: $testname"
-                docker run --name garage_benchmark -e MJKEY="$(cat ~/.mujoco/mjkey.txt)" rlworkgroup/garage-headless nose2 -c setup.cfg tests.benchmarks.$testname
+                docker run --name garage_benchmark -e MJKEY="$(cat ~/.mujoco/mjkey.txt)" rlworkgroup/garage-headless pytest ./tests/benchmarks/$testname.py
```

Comment:
hmm can you try using the garage makefile for this? it should be as simple as `make run-headless RUN_CMD="pytest tests/benchmarks/$testname.py"`